gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy/src/main/java/io/gravitee/plugin/endpoint/http/proxy/connector/HttpConnector.java

```diff
@@ -126,9 +127,17 @@ public Completable connect(final HttpExecutionContext ctx) {
                 .map(this::customizeHttpClientRequest)
                 .flatMap(httpClientRequest -> {
                     observableHttpClientRequest.httpClientRequest(httpClientRequest.getDelegate());
-                    return httpClientRequest.rxSend(
-                        request.chunks().map(buffer -> io.vertx.rxjava3.core.buffer.Buffer.buffer(buffer.getNativeBuffer()))
-                    );
+                    if ("chunked".equals(request.headers().get("transfer-encoding"))) {
```

Comment:
I'm wondering if we could have a different case for the value and/or the header name? I'm hoping we deal with that earlier (by forcing lowercase). Did you test this case?